inferior-idris.el

```diff
@@ -335,6 +335,9 @@ versions cannot deal with that."
 ignoring intermediate output. If `NO-ERRORS' is non-nil, don't
 trigger warning buffers and don't call `ERROR' if there was an
 Idris error."
+  (message "Evaluating %s (no-errors: %s)"
```

Comment:
This seems like temporary debugging code that was left in by mistake.

Reply:
Oh, I thought I had removed that!